src/network/networkd-link.h

```diff
@@ -118,6 +118,8 @@ typedef struct Link {
         Set *ndisc_rdnss;
         Set *ndisc_dnssl;
 
+        sd_radv *radv;
```

Comment:
this PR should also contain code that destroy this object when the link itself is destroyed?